Makefile

```diff
@@ -1,3 +1,18 @@
+# The following parameters are required:
+# - AWS_DEFAULT_REGION
+# - AWS_ACCESS_KEY
+# - AWS_SECRET_ACCESS_KEY
+# - GITHUB_ORG
+# - GITHUB_REPO
+# - CIRCLECI_TOKEN
+#
 
 run:
-	@echo Done
+	@echo "Set CIRCLECI environment variables\n"
+	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_DEFAULT_REGION", "value":"${AWS_DEFAULT_REGION}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_TOKEN}
```

Comment:
These aren't quite the right env vars for this project. In the deploying with Zero doc it lists the env vars as:
```
AWS_ACCESS_KEY_ID
AWS_SECRET_ACCESS_KEY
CIRCLECI_API_KEY
```

Reply:
We determined the env vars needed from the `setup` command in the `aws-cli` orb: https://circleci.com/orbs/registry/orb/circleci/aws-cli#commands-setup

I don't think CIRCLECI_API_KEY is needed is an env var, we just use it to make the api call.

Reply:
It's required for the queueing orb.

Reply:
Also it should be `AWS_ACCESS_KEY_ID`

Reply:
Plus you don't need the region, that will be supplied in the circle config.

Reply:
The google doc is the right place to look for this info.

Reply:
Queuing orb will need you to allow 3rd party apps as a Github Admin in the CircleCI settings. No way to do this via the API.